docfx.json

```diff
@@ -157,72 +164,27 @@
         },
         "fileMetadata": {
             "feedback_system": {
-                "_csharplang/**.*": "OpenSource",
-                "_csharpstandard/standard/*.md": "OpenSource",
-                "_vblang/**.*": "OpenSource",
-                "_roslyn/**.*": "OpenSource",
-                "docs/**.*": "OpenSource",
                 "docs/standard/design-guidelines/**/**.md": "None",
                 "docs/framework/data/adonet/**/**.md": "None",
                 "docs/framework/data/wcf/**/**.md": "None",
                 "docs/framework/ui-automation/**/**.md": "None",
                 "docs/framework/wcf/**/**.md": "None"
             },
-            "open_source_feedback_contributorGuideUrl": {
-                "docs/**/**.*": "https://learn.microsoft.com/contribute/content/dotnet/dotnet-contribute",
-                "_csharplang/**.*": "https://learn.microsoft.com/contribute/content/dotnet/dotnet-contribute",
-                "_csharpstandard/standard/*.md": "https://learn.microsoft.com/contribute/content/dotnet/dotnet-contribute",
-                "_vblang/**.*": "https://learn.microsoft.com/contribute/content/dotnet/dotnet-contribute",
-                "_roslyn/**.*": "https://learn.microsoft.com/contribute/content/dotnet/dotnet-contribute"
-            },
             "open_source_feedback_issueUrl": {
-                "docs/**.*": "https://github.com/dotnet/docs/issues/new?template=customer-feedback.yml",
                 "_csharplang/**.*": "https://github.com/dotnet/charplang/issues/new?template=docs-feedback.yml",
-                "_csharpstandard/standard/*.md": "https://github.com/dotnet/csharpstandard/issues/new?template=docs-feedback.yml",
-                "_vblang/**.*": "https://github.com/dotnet/docs/issues/new?template=customer-feedback.yml",
-                "_roslyn/**.*": "https://github.com/dotnet/docs/issues/new?template=customer-feedback.yml"
+                "_csharpstandard/standard/*.md": "https://github.com/dotnet/csharpstandard/issues/new?template=docs-feedback.yml"
             },
             "open_source_feedback_productName": {
-                "docs/**.*": ".NET",
                 "_csharplang/**.*": "C# feature specification",
                 "_csharpstandard/standard/*.md": "C# Standard documentation",
                 "_vblang/**.*": "Visual Basic language spec",
                 "_roslyn/**.*": "Roslyn breaking changes"
             },
-            "open_source_feedback_productDescription": {
```

Comment:
This node was removed intentionally. The productDescription is hardcoded now to facilitate localization.